symengine/parser.cpp

```diff
@@ -196,6 +208,31 @@ class ExpressionParser
                                  parse_string(iter + 1, operator_end[iter]));
                     iter = operator_end[iter] - 1;
 
+                } else if (s[iter] == '=') {
+                    result = Eq(result,
+                                parse_string(iter + 1, operator_end[iter]));
+                    iter = operator_end[iter] - 1;
+
+                } else if (s[iter] == '<' and s[iter + 1] == '=') {
```

Comment:
I have my doubts about that as well. A hacky alternative that occurred to me was to use other symbols, such as `#` or `@` for replacing all the instances of `<=` and `>=`, just like it's being done for `**` to `^` during preprocessing. What would you suggest?